src/tests/editor.test.js

```diff
@@ -1748,11 +1748,11 @@ describe('Editor', () => {
                     contentAfter: '<p>ab[]</p><p t="unbreak">def</p>',
                 });
             });
-            it('should delete first character of unbreakable, ignoring selected paragraph break', async () => {
+            it('should not delete across an unbreakable', async () => {
                 await testEditor(BasicEditor, {
                     contentBefore: '<p>abc[</p><p t="unbreak">d]ef</p>',
                     stepFunction: deleteBackward,
-                    contentAfter: '<p>abc[]</p><p t="unbreak">ef</p>',
+                    contentAfter: '<p>abc[]</p><p t="unbreak">def</p>',
```

Comment:
I disagree with this as much as I disagree with the new `table` behavior, which are basically the same.

Reply:
Same reply ^^